docusaurus.config.js

```diff
@@ -74,24 +74,24 @@ const config = {
         },
       },
     ],
-    [
-      "docusaurus-plugin-typedoc",
-      {
-        id: "akasha-ui-hooks",
-        entryPoints: ["./akasha-core/ui/hooks/src/index.ts"],
-        entryPointStrategy: "expand",
-        tsconfig: "./akasha-core/ui/hooks/tsconfig.json",
-        readme: "docs/react-hooks/_hooks-list_.md",
-        out: "./react-hooks",
-        name: "React hooks",
-        excludeInternal: true,
-        sidebar: {
-          categoryLabel: "React hooks",
-          position: 5,
-        },
-        frontmatter: null,
-      },
-    ],
+    // [
+    //   "docusaurus-plugin-typedoc",
```

Comment:
why is this commented? :thinking:

Reply:
I commented this out because I created the documentation for the newly generated hooks here right in the docs folder and not extracting anything from the akasha-core repo since the new-hook file there was auto-generated and doesn't contain any documentation.